server/db/restaurantData.js

```diff
@@ -0,0 +1,404 @@
+const _restaurant_name_id = [   
+    {
```

Comment:
[check indentation style guide ](https://github.com/airbnb/javascript#whitespace--spaces)

Reply:
indentations are fixed in the next PR